### PR TITLE
[Issue-5]: Fix Header

### LINKED ISSUE #5: Header Sign-in functionality
Update header to match caesars.com/careers/ site

### PR DESCRIPTION
**Pull request by swetabar**
Fix #5 

## 🔗 Test URLs:

- Before: https://main--caesars-careers--hlxsites.hlx.page/nav
- After: https://issue-5-fix-header--caesars-careers--hlxsites.hlx.page/nav

## 📝 Description:

Screen Shot 2023-05-08 at 11 55 10 AM

Screen Shot 2023-05-08 at 11 55 33 AM

### CODE REVIEW

**Comment by tmathern** [comment]
I guess seeing the content being duplicated is normal?
Screenshot 2023-05-08 at 9 02 06 AM

File: careers/blocks/header/header.css [hunk 1]
```diff
@@ -442,6 +437,9 @@ header nav .nav-tools p:first-of-type a:hover {
 
   header .global-nav-desktop ul li a:hover {
     text-decoration: none;
+    transform: scaleX(0);
```
Comment (tmathern) [inline careers/blocks/header/header.css:440]: Why do we need that?

File: careers/blocks/header/header.css [hunk 2]
```diff
@@ -442,6 +437,9 @@ header nav .nav-tools p:first-of-type a:hover {
...
+    transition: transform .15s cubic-bezier(0.4,0,0.2,1);
+    background: #fff;
```
Comment (tmathern) [inline careers/blocks/header/header.css:442]: Use a var?

**Comment by swetabar** [comment]
> I guess seeing the content being duplicated is normal for now? Screenshot 2023-05-08 at 9 02 06 AM

Ah yes. That's the nav page. We could see the nav bar on any other page as well. Then the content won't be duplicated.
https://issue-5-fix-header--caesars-careers--hlxsites.hlx.page/footer

File: careers/styles/styles.css [hunk 3]
```diff
@@ -211,13 +211,13 @@ button {
 .button-container > em > a.secondary,
 button {
   min-width: 133px;
-  border: 1px solid var(--static-color-black);
+  border: 1px solid var(--static-color-pink-red);
```
Comment (amol-anand) [inline careers/styles/styles.css:214]: is the pink-red color for buttons global across the site?

File: careers/blocks/header/header.css (hunk 1 above)
Comment (swetabar) [reply careers/blocks/header/header.css]: We don't need it. Removed it!

File: careers/blocks/header/header.css (hunk 2 above)
Comment (swetabar) [reply careers/blocks/header/header.css]: Done.

File: careers/blocks/header/header.js [hunk 4]
```diff
@@ -377,13 +329,15 @@ export default async function decorate(block) {
...
-    toggleMenu(nav, navSections, screenConfig.smallDesktop.media.matches);
+    toggleMenu(nav, navSections, screenConfig.smallDesktop.media.matches
+      || screenConfig.largeDesktop.media.matches);
 
     // add event listeners when window width changes
     screenConfig.smallDesktop.media.addEventListener('change', () => {
       const localNavTitle = block.querySelector('.local-nav-title');
       const globalNavTitle = block.querySelector('.global-nav-title');
-      toggleMenu(nav, navSections, screenConfig.smallDesktop.media.matches);
+      toggleMenu(nav, navSections, screenConfig.smallDesktop.media.matches
```
Comment (amol-anand) [inline careers/blocks/header/header.js:339]: Why do we need this? The CSS should just handle hiding / showing the mobile nav vs regular nav based on media queries.

**Comment by amol-anand** [comment]
Please don't merge yet. I will take a look to see if I can get rid of some more stuff.

File: careers/styles/styles.css (hunk 3 above)
Comment (swetabar) [reply careers/styles/styles.css:214]: Yes. Looks like all the buttons on caesars-careers have a pink-red colour.

Comment (tmathern) [reply careers/styles/styles.css:214]: +1

**Pull request merged**